EIPS/eip-681.md

```diff
@@ -0,0 +1,78 @@
+
+This specification supersedes ERC #67, which is a URL format for representing arbitrary transactions in a low-level fashion. This ERC focuses specifically on the important special case of payment requests, while allowing for other, ABI-specified transactions.
+
+## Specification
+
+### Syntax
+Payment request URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" target_address [ "@" chain_id ] [ "/" function_name ] [ "?" parameters ]
+    target_address          = ethereum_address
+    chain_id                = 1*DIGIT
+    function_name           = STRING
+    ethereum_address        = ( "0x" 40*40HEXDIG ) / ENS_NAME
+    parameters              = parameter *( "&" parameter )
+    parameter               = key "=" value
+    key                     = "value" / "gas" / TYPE
+    value                   = number / ethereum_address / STRING
+    number                  = [ "-" / "+" ] *DIGIT [ "." 1*DIGIT ] [ ( "e" / "E" ) [ 1*DIGIT ] [ "+" UNIT ]
+
+
+Where `TYPE` is a standard ABI type name, as defined in [Ethereum Contract ABI specification](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI). `STRING` is a URL-encoded unicode string of arbitrary length, where delimiters and the 
+percentage symbol (`%`) are mandatorily hex-encoded with a `%` prefix.
+
+`UNIT` is a URL-encoded unicode string. If `UNIT` is ETH, it always means a multiplier of 10<sup>18</sup>. If it is something 
+else AND the addressed contract has a `symbol` field exactly matching this string AND the contract has a `decimals` field, then 
+10 to that power is used as a multiplier. Otherwise, the payment request is deemed invalid. Applications that have no access to 
+the blockchain should refuse accepting requests with a non-empty `UNIT`, if it is not ETH.
+
+Note that a `number` can be expressed in *scientific notation*, with a multiplier of a power of 10. The use of this notation is strongly encouraged when expressing monetary value in Ethers or ERC #20 tokens in atomic units (e. g. Wei, in case of Ether).
+
+If *key* in the parameter list is `value` or `gas` then *value* MUST be a `number`. Otherwise, it must correspond to the `TYPE` string used as *key*.
+
+For the syntax of ENS_NAME, please consult ERC #137 defining Ethereum Name Service.
+
+### Semantics
+
+`target_address` is mandatory and denotes either the beneficiary of native token payment (see below) or the contract address with which the user is asked to interact.
+
+`chain_id` is optional and contains the decimal chain ID, such that transactions on various test- and private networks can be requested. If no `chain_id` is present, the main network (1) is assumed.
```

Comment:
Unfortunately this breaks backward-compatibility with ERC-67 - would love if this could be changed to :

> If no chain_id` is present, the current network is used.